nixos/tests/phylactery.nix

```diff
@@ -0,0 +1,19 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }: {
```

Comment:
Separate this into its own commit. `nixos/tests: add phlactery`

Reply:
Done!